src/main/java/org/eclipse/tractusx/ssi/lib/jwt/SignedJwtFactory.java

```diff
@@ -130,15 +131,17 @@ public SignedJWT create(
     final String issuer = didIssuer.toString();
     final String subject = holderIssuer.toString();
 
-    vc.remove(VerifiableCredential.PROOF);
+    final Date issueDate = Date.from(Instant.parse((String) vc.get("issuanceDate")));
+
+    vc.remove(Verifiable.PROOF);
 
     var claimsSet =
         new JWTClaimsSet.Builder()
             .issuer(issuer)
             .subject(subject)
             .claim("vc", vc)
             .expirationTime(expDate)
-            .jwtID(UUID.randomUUID().toString())
+            .issueTime(issueDate)
```

Comment:
why jwtID is removed?

In the case of VP as JWT, the issue time should be the current time. 
This is a common method used to create VC and VP

Reply:
because we don't want the payload of the jwt to be changed